### PR TITLE
Base css

### PR DESCRIPTION
**Pull request by zilva149**
base.css file, css reset

### CODE REVIEW

**Review by EvgenySivaykin — DISMISSED** [review]
OK

**Review by Dorenus — DISMISSED** [review]
Everything seems good.

**Review by Rasosm — APPROVED** [review]

File: css/base/base.css [hunk 1]
```diff
@@ -1,13 +1,126 @@
...
+    --pink: #e83e8c;
+    --red: #dc3545;
+    --orange: #fd7e14;
+    --yellow: #ffc107;
+    --green: #28a745;
+    --teal: #20c997;
+    --cyan: #17a2b8;
+    --white: #fff;
+    --gray: #6c757d;
+    --gray-dark: #343a40;
```
Comment (liepsneletech) [inline css/base/base.css:19]: Maybe --dgray for consistency? :)

**Review by liepsneletech — COMMENTED** [review]
Everything looks fine, just small fix : )

File: css/base/base.css (hunk 1 above)
Comment (zilva149) [reply css/base/base.css:19]: ok, will change it in a minute

**Review by liepsneletech — APPROVED** [review]

**Review by EvgenySivaykin — APPROVED** [review]

**Review by Rasosm — APPROVED** [review]
Good:)

**Pull request merged**